tests/integration/docker_test/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+
+services:
+  chrome:
+    container_name: chrome_driver
+    build:
+      context: .
+      dockerfile: chrome.dockerfile
+    volumes:
+      - ../../../src:/var/www/docker-test/src
+      - ../../../tests:/var/www/docker-test/tests
+      - ../../../deps.ts:/var/www/docker-test/deps.ts
+      - ../../../mod.ts:/var/www/docker-test/mod.ts
```

Comment:
Why 4 separate volumes and not something like .:/var/www/docker-test?

I think in the docker image, we can make the working dir at the top level so this doesn't have to go back 3 levels

Reply:
Not really sure how we’d go about that as `.` would be `.../sinco/tests/integration/Docker-test`

Reply:
You're right 😄 i didn't realize that on mobile 😬